Aquality.Appium.Mobile/tests/Aquality.Appium.Mobile.Tests/Samples/Android/NativeApp/ApiDemos/Screens/ViewControlsScreen.cs

```diff
@@ -1,17 +1,26 @@
-﻿using Aquality.Appium.Mobile.Elements.Interfaces;
+﻿using Aquality.Appium.Mobile.Actions;
+using Aquality.Appium.Mobile.Elements.Interfaces;
 using OpenQA.Selenium;
 using OpenQA.Selenium.Appium;
 
 namespace Aquality.Appium.Mobile.Tests.Samples.Android.NativeApp.ApiDemos.Screens
 {
     public class ViewControlsScreen : ApplicationActivityScreen
     {
+        private readonly ILabel lblAllInsideScrollView = ElementFactory.GetLabel(
+            MobileBy.AccessibilityId("(And all inside of a ScrollView!)"),
+            "All inside of Scroll View");
+
+        private readonly IButton btnDisabled = ElementFactory.GetButton(
```

Comment:
we used to call fields in full form in .NET libraries, like DisabledButton, AllInsideScrollViewLabel and so on